Taskfile.yml

```diff
@@ -32,12 +33,18 @@ tasks:
     cmds:
       - go install github.com/golangci/golangci-lint/cmd/golangci-lint@{{.GOLANGCI_VERSION}}
       - go install gotest.tools/gotestsum@{{.GOTESTSUM_VERSION}}
+      - go install github.com/segmentio/golines@{{.GOLINES_VERSION}}
 
   lint:
     desc: Lint the code.
     cmds:
       - golangci-lint run ./...
 
+  lint:linelength:
+    desc: Enforce max line length. Use the tool output only as a hint, sometimes it makes wrong decisions.
+    cmds:
+      - golines --max-len=90 --write-output .
```

Comment:
For python code, we use 99. I think it would be simpler to have the same limit independently of the language.

Reply:
I have to agree I made the error of accepting the 99...
OK, I will redo this thing then.

Reply:
Thanks!

Reply:
Done (force push)